easybuild/easyconfigs/i/IsoQuant/IsoQuant-3.3.0-foss-2022b.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'Tarball'
+
+name = 'IsoQuant'
+version = '3.3.0'
+
+homepage = 'https://github.com/ablab/IsoQuant'
+description = """IsoQuant is a tool for the genome-based analysis of long RNA reads,
+ such as PacBio or Oxford Nanopores. IsoQuant allows to reconstruct and quantify
+ transcript models with high precision and decent recall. If the reference annotation is given,
+ IsoQuant also assigns reads to the annotated isoforms based on their intron and exon structure.
+ IsoQuant further performs annotated gene, isoform, exon and intron quantification.
+ If reads are grouped (e.g. according to cell type), counts are reported according to the provided grouping."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/ablab/%(name)s/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}]
+
+checksums = ['5c0e793a33f1b39b9f4888f7bade7f827f68bfb113535e08099906f7ad4689ce']
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('gffutils', '0.12'),
+    ('Biopython', '1.81'),
+    ('SciPy-bundle', '2023.02'),
+    ('pybedtools', '0.9.0'),
+    ('Pysam', '0.21.0'),
+    ('pyfaidx', '0.7.2.1'),
+    ('minimap2', '2.26'),
+    ('SAMtools', '1.17'),
+]
+
+modextrapaths = {
+    'PATH': '',
+    'PYTHONPATH': '',
+}
+
+sanity_check_paths = {
+    'files': ['isoquant.py'],
+    'dirs': [],
+}
+
+sanity_check_commands = ["isoquant.py --test"]
```

Need to include a `cd` command here to avoid that this check fails when using `--sanity-check-only` or `--module-only` and `--read-only-installdir` was used for the installation.

```suggestion
sanity_check_commands = ["cd %(builddir)s && isoquant.py --test"]